docs/content/docs/integrations/nuxt.mdx

```diff
@@ -68,6 +68,27 @@ const session = authClient.useSession()
 </template>
 ```
 
+### Server Usage
+
+The `api` object exported from the auth instance contains all the actions that you can perform on the server. Every endpoint made inside Better Auth is a invocable as a function. Including plugins endpoints.
+
+**Example: Getting Session on a server API route**
+
+```tsx title="server/api/example.ts"
```

Comment:
Thanks for the quick merge but `tsx` should actually be `ts`, sorry 

@Bekacru